### PR TITLE
[MFTF] Use action group for logout

### PR DESCRIPTION
**Pull request by Usik2203**
Use `<actionGroup ref="AdminLogoutActionGroup" stepKey="amOnLogoutPage"/>` for logout instead 
`<amOnPage url="{{AdminLogoutPage.url}}" stepKey="amOnLogoutPage"/>`

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Nice catch! I remember that I've done it once or twice, but there are still leftovers, as we can see. Appreciate your efforts to get rid of such.

File: app/code/Magento/Bundle/Test/Mftf/Test/CurrencyChangingBundleProductInCartTest.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                 <click selector="{{CurrencySetupSection.currencyOptions}}" stepKey="closeOptions"/>
                 <waitForPageLoad stepKey="waitForCloseOptions"/>
                 <click stepKey="saveUnselectedConfigs" selector="{{AdminConfigSection.saveButton}}"/>
-                <amOnPage url="{{AdminLogoutPage.url}}" stepKey="logout"/>
+                <actionGroup ref="AdminLogoutActionGroup" stepKey="amOnLogoutPage"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Bundle/Test/Mftf/Test/CurrencyChangingBundleProductInCartTest.xml]: If you can keep the original `stepKey` that would be appreciated, as some developers might have attached their Actions to it.

File: app/code/Magento/Bundle/Test/Mftf/Test/BundleProductWithTierPriceInCartTest.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@
             <actionGroup ref="AdminDeleteCustomerActionGroup" stepKey="deleteCustomer">
                 <argument name="customerEmail" value="CustomerEntityOne.email"/>
             </actionGroup>
-            <amOnPage url="{{AdminLogoutPage.url}}" stepKey="logout"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="amOnLogoutPage"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Bundle/Test/Mftf/Test/BundleProductWithTierPriceInCartTest.xml]: If you can keep the original `stepKey` that would be appreciated, as some developers might have attached their Actions to it.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7767](https://jira.corp.magento.com/browse/ENGCOM-7767) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by lbajsarowicz** [comment]
@magento run Functional Tests EE

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

File: app/code/Magento/Bundle/Test/Mftf/Test/CurrencyChangingBundleProductInCartTest.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Bundle/Test/Mftf/Test/CurrencyChangingBundleProductInCartTest.xml]: **Done**

File: app/code/Magento/Bundle/Test/Mftf/Test/BundleProductWithTierPriceInCartTest.xml (hunk 2 above)
Comment (Usik2203) [reply app/code/Magento/Bundle/Test/Mftf/Test/BundleProductWithTierPriceInCartTest.xml]: **Done**

**Comment by Usik2203** [comment]
@magento run Functional Tests EE

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**